### PR TITLE
V1 update_feature

### PR DESCRIPTION
**Pull request by matijamjam**
#first github message - Update feature with :
- controller : def edit, def update, def boat_param. 
- show.html : edit link
- edit.html : simple_form

### CODE REVIEW

**Comment by GregPoth** [comment]
Hi Mathilda, I'll review if you will

File: app/controllers/boats_controller.rb [hunk 1]
```diff
@@ -1,2 +1,11 @@
 class BoatsController < ApplicationController
+  def edit
+    @boat = Boat.find(params)
```
Comment (GregPoth) [inline app/controllers/boats_controller.rb]: Shouldn't you write :
@boat = Boat.find(params[:id])

File: app/controllers/boats_controller.rb [hunk 2]
```diff
@@ -1,2 +1,11 @@
...
+  end
+
+  def update
+    @boat = Boat.find(params[:id])
```
Comment (GregPoth) [inline app/controllers/boats_controller.rb]: as you use it twice you should maybe write a def for this and before action it :
before_action: only [:edit, :update]

private
def find_boat
      @boat = Boat.find(params[:id])
end

File: app/controllers/boats_controller.rb (hunk 1 above)
Comment (Zephyress) [inline app/controllers/boats_controller.rb]: params[:id]

File: app/views/pages/edit.html.erb [hunk 3]
```diff
@@ -0,0 +1,7 @@
+%= simple_form_for(boat) do |f| %>
+  <%= f.input :title %>
+  <%= f.input :description %>
+  <%= f.input :category %>
+  <%= f.input :price %>
```
Comment (GregPoth) [inline app/views/pages/edit.html.erb:5]: I would add the feature :
  <%= f.input :start_date %>
  <%= f.input :end_date %>

File: app/controllers/boats_controller.rb [hunk 4]
```diff
@@ -1,2 +1,11 @@
...
+    @boat.update(task_params)
```
Comment (Zephyress) [inline app/controllers/boats_controller.rb]: Pas task_params mais boat_params

**Review by GregPoth — COMMENTED** [review]
Looks good for a first draft

**Review by Zephyress — CHANGES_REQUESTED** [review]

**Pull request merged**